sway/input/cursor.c

```diff
@@ -449,15 +449,17 @@ static void dispatch_cursor_button_floating(struct sway_cursor *cursor,
 	bool over_title = edge == WLR_EDGE_NONE && !surface;
 
 	// Check for beginning move
-	if (button == BTN_LEFT && state == WLR_BUTTON_PRESSED &&
+	uint32_t btn_move = config->floating_mod_inverse ? BTN_RIGHT : BTN_LEFT;
+	if (button == btn_move && state == WLR_BUTTON_PRESSED &&
 			(mod_pressed || over_title)) {
-		seat_begin_move(seat, cont, BTN_LEFT);
+		seat_begin_move(seat, cont, btn_move);
```

Comment:
Nitpick: You pass `button` to `seat_begin_resize` below, so for consistency you should pass `button` here despite it being equal to `btn_move`.